### PR TITLE
Facebook data correction and new content

### LINKED ISSUE #45: facebook_last_location

### PR DESCRIPTION
**Pull request by GawenMonnot**
fixes #45 #44 #43

### CODE REVIEW

File: src/activities/facebook/primary_public_location.rs [hunk 1]
```diff
@@ -14,7 +14,7 @@ pub struct PrimaryPublicLocationObjects {
 #[allow(non_snake_case)]
 #[derive(Deserialize, Debug)]
 pub struct PrimaryPublicLocation {
-  pub primary_public_location: PrimaryPublicLocationObjects
+  pub primary_public_location_v2: PrimaryPublicLocationObjects
```
Comment (chansuke) [inline src/activities/facebook/primary_public_location.rs:17]: I appreciate your work!!! `v2` is meaning a new version of the primary public location?

File: src/main.rs [hunk 2]
```diff
@@ -123,7 +125,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 
             let response = result.saveToDb(&conn)?;
 
-            println!("( {} records )", result.phone_number_location.len());
+            println!("( {} records )", 1);
```
Comment (chansuke) [inline src/main.rs]: It always printing a fixed value.

File: src/activities/facebook/primary_public_location.rs (hunk 1 above)
Comment (dkastl) [reply src/activities/facebook/primary_public_location.rs:17]: Yes. If you request your data now, the format has changed and `_v2` is aappended.

File: src/main.rs (hunk 2 above)
Comment (GawenMonnot) [reply src/main.rs]: Because there is only one content in the file. Am I wrong ?

Comment (chansuke) [reply src/main.rs]: @GawenMonnot Thanks for your answer. You can print like this ` println!("1 records")` and if the result is always one content, you don't need to print out IMHO.

Comment (GawenMonnot) [reply src/main.rs]: Yes you are right. Thank you!

File: src/main.rs [hunk 3]
```diff
@@ -125,7 +125,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 
             let response = result.saveToDb(&conn)?;
 
-            println!("( {} records )", 1);
+             println!("( 1 record )");
```
Comment (chansuke) [inline src/main.rs:128]: thanks for your change.

**Review by chansuke — APPROVED** [review]
@GawenMonnot Thank you for your work. You can squash the commits and merge.

**Pull request merged**